src/snakemake/runtime_dependencies.py

```
@@ -0,0 +1,223 @@
+            pkg_str = line.removeprefix(install_prefix)
+            yield Package.from_str(pkg_str)
+
+
+def get_packages_in_prefix(prefix: Path) -> Iterable[Package]:
+    """Retrieves the packages installed under the given prefix using
+    `uv pip list --prefix <prefix>`.
+    """
+    return map(
+        Package,
+        json.loads(
+            sp.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)